package.json

```diff
@@ -14,6 +14,7 @@
     "@radix-ui/react-slot": "^1.1.0",
     "@vercel/analytics": "^1.3.2",
     "class-variance-authority": "^0.7.0",
+    "mermaid": "^11.4.0",
```

Comment:
Is this being used?

Reply:
I got some errors when i tried to run `npm run dev` without it.